### PR TITLE
french translation

### PR DESCRIPTION
**Pull request by Super-Baleine**

### CODE REVIEW

File: lximage-qt/lximage-qt_fr.ts [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    <message>
+        <location filename="../../mainwindow.cpp" line="819"/>
+        <source>Thumbnails</source>
+        <translation>Miniatures</translation>
+    </message>
+</context>
+<context>
+    <name>MainWindow</name>
+    <message>
+        <location filename="../../../build/src/ui_mainwindow.h" line="412"/>
```
Comment (palinek) [inline lximage-qt/lximage-qt_fr.ts:84]: There are stil the ../../../build/src/ui_*.h and not the ../../*.ui filenames

Comment (Super-Baleine) [inline lximage-qt/lximage-qt_fr.ts:84]: yea, wait

**Comment by Super-Baleine** [comment]
Also, can you tell me how to squash ? Because it didn't work before, what i found on the net

**Comment by palinek** [comment]
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
`git rebase -i origin/master` (and do with the commits what is needed)

**Pull request closed without merging**